src/main/java/io/spring/ge/conventions/gradle/BuildScanConventions.java

```diff
@@ -108,6 +108,10 @@ private boolean isBamboo() {
 		return this.env.containsKey(BAMBOO_RESULTS_ENV_VAR);
 	}
 
+	private boolean isCircleCi() {
+		return this.env.containsKey("CIRCLECI");
+	}
+
 	private boolean isConcourse() {
 		return this.env.containsKey("CI");
```

Comment:
CircleCI also sets an env var `CI=true` on its builds. I'm not sure if there is a more discerning env var for Concourse. Right now it is not practically an issue since CircleCI is checked before Concourse, but it does feel fragile to potential future changes in this code.

Reply:
For other CI systems that we support in the Common Custom User Data Gradle plugin, we detect the presence of the URL environment variable like so: https://github.com/gradle/gradle-enterprise-build-config-samples/blob/master/common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanEnhancements.java#L175